<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SellerTorrentPluginConfigurationDialog</class>
 <widget class="QDialog" name="SellerTorrentPluginConfigurationDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>347</width>
    <height>104</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Seller</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>146</x>
     <y>50</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="minPriceLabel">
   <property name="geometry">
    <rect>
     <x>22</x>
     <y>24</y>
     <width>231</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Price/piece (*):</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="minPriceLineEdit">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>20</y>
     <width>57</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QLabel" name="minLockLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>234</y>
     <width>109</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Min. Lock time (h:m):</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="minFeeLineEdit">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>264</y>
     <width>57</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>0.1</string>
   </property>
  </widget>
  <widget class="QLabel" name="minFeeLabel">
   <property name="geometry">
    <rect>
     <x>26</x>
     <y>268</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Min. fee/kB (m…É):</string>
   </property>
  </widget>
  <widget class="QTimeEdit" name="minLockTimeEdit">
   <property name="geometry">
    <rect>
     <x>164</x>
     <y>230</y>
     <width>57</width>
     <height>22</height>
    </rect>
   </property>
   <property name="time">
    <time>
     <hour>3</hour>
     <minute>0</minute>
     <second>0</second>
    </time>
   </property>
  </widget>
  <widget class="QLabel" name="maxNumberOfSellersLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>200</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Maximum #sellers:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="maxNumberOfSellersLineEdit">
   <property name="geometry">
    <rect>
     <x>164</x>
     <y>198</y>
     <width>57</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QLabel" name="maxConfirmationTimeLabel">
   <property name="geometry">
    <rect>
     <x>26</x>
     <y>292</y>
     <width>139</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Max. conf. time (h:m):</string>
   </property>
  </widget>
  <widget class="QTimeEdit" name="maxConfirmationTimeTimeEdit">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>290</y>
     <width>57</width>
     <height>22</height>
    </rect>
   </property>
   <property name="time">
    <time>
     <hour>0</hour>
     <minute>15</minute>
     <second>0</second>
    </time>
   </property>
  </widget>
  <widget class="QLabel" name="computedSatoshiesMinPrice">
   <property name="geometry">
    <rect>
     <x>22</x>
     <y>160</y>
     <width>231</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Price/piece (satoshies):</string>
   </property>
  </widget>
  <widget class="QLabel" name="computedSatoshiesMinPriceValue">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>160</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>N/A</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SellerTorrentPluginConfigurationDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SellerTorrentPluginConfigurationDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
